src/routes/components/TextField.tsx

```diff
@@ -438,16 +448,16 @@ export default function TextFieldPage() {
                 tags="react"
                 allowCopy={true}
                 code={`
-                const [value, setValue] = useState<string>("");
+                const [theValue, setTheValue] = useState<string>("");
               
                 onChange(name: string, value: string) {
-                  setValue(value);
+                  setTheValue(value);
                 }
               `}
               />
 
               <GoAFormItem {...formItemProps}>
-                <GoAInput {...componentProps} onChange={noop} />
+                <GoAInput {...componentProps} name="item" value="" onChange={onChange}/>
```

Comment:
In some places the useState is removed, but it is modified here. Is there a reason for that?